config/checkstyle_non_main_files_checks.xml

```diff
@@ -23,6 +23,11 @@
                value="^(?!(.*href=|.*http(s)?:|import |(.* )?package |.* files=|.*\.dtd)).{101,}$"/>
       <property name="message" value="Line should not be longer than 100 symbols"/>
     </module>
+    <!--This check is to enforce -e for all maven commands -->
```

Comment:
this comment is not required ...
BUT `<!--This` .... there is missed space and non of our checks complain about this, sad.
As you are good with regexp .... can you make regexp to validate missed space in comments ? I thought that we already do this validation, but I might forgot something.

Reply:
ok, my pleasure

Reply:
@HuGanghui , please create issue on this and share here a link

Reply:
link is here, Making regex to validate missed space in comments in non main files #8169